src/Orleans/packages.config

```diff
@@ -1,10 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="log4net" version="2.0.3" targetFramework="net45" />
   <package id="Microsoft.Data.Edm" version="5.6.0" targetFramework="net45" />
   <package id="Microsoft.Data.OData" version="5.6.0" targetFramework="net45" />
   <package id="Microsoft.Data.Services.Client" version="5.6.0" targetFramework="net45" />
   <package id="Microsoft.WindowsAzure.ConfigurationManager" version="2.0.0" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="5.0.8" targetFramework="net45" />
   <package id="System.Spatial" version="5.6.0" targetFramework="net45" />
   <package id="WindowsAzure.Storage" version="4.2.0" targetFramework="net45" />
   <package id="WiX.Toolset" version="3.9.1006.0" targetFramework="net4000" />
+  <package id="ZooKeeperNetEx" version="3.4.6-alpha12" targetFramework="net45" />
```

Comment:
We need to load this dependency dynamically. We are going to do the same for the dependency on Azure for the Azure Table-based membership.